enclave_wrangler/objects_api.py

```diff
@@ -671,10 +671,10 @@ def concept_set_members__cset_rows_to_db(con: Connection, cset: Dict, members: L
         'codeset_id': cset['codesetId'],
         'concept_id': member['conceptId'],
         'concept_set_name': cset['conceptSetNameOMOP'],
-        'is_most_recent_version': cset['isMostRecentVersion'], # not sure: is this correct?
```

Comment:
I deleted 3 comments that said `# not sure: is this correct?`, as I think I was doing this correctly. It's been almost a year since I've done this and we've had no problems yet, so.